packages/cosmic-swingset/test/test-home.js

```diff
@@ -102,6 +102,46 @@ test.serial('home.wallet - receive zoe invite', async t => {
   );
 });
 
+test('home.localTimerService createNotifier', async t => {
+  const { localTimerService } = E.G(home);
+  const notifier = E(localTimerService).createNotifier(1, 1);
+  const update1 = await E(notifier).getUpdateSince();
+  t.is(update1.updateCount, 2);
+  const update2 = await E(notifier).getUpdateSince(update1.updateCount);
+  t.is(update2.updateCount, 3);
+  t.truthy(update2.value > update1.value);
```

Comment:
Can we add a comment about the actual value being a timestamp, and therefore we can't test against it directly, only relatively?